src/autogluon/bench/frameworks/tabular/tabular_benchmark.py

```diff
@@ -11,6 +11,25 @@
 
 
 class TabularBenchmark(Benchmark):
+    """A benchmark class for AutoGluon Tabular.
+
+    Args:
+        benchmark_name (str): The name of the benchmark.
+        root_dir (str): The root directory for storing benchmark results (default: "./benchmark_runs/tabular/").
```

Comment:
My understanding is technically constructor args should be documented in the `__init__` docstring. No problem if you disagree.